Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/scan/ScanCheckInQrCodeFragment.kt

```diff
@@ -95,9 +95,11 @@ class ScanCheckInQrCodeFragment :
         ) {
             if (shouldShowRequestPermissionRationale(Manifest.permission.CAMERA)) {
                 showCameraPermissionRationaleDialog()
+                viewModel.setCameraDeniedPermanently(false)
```

Comment:
When going to system app settings and changing it to "ask every time" or "allow while app is open", after returning to the CWA, the camera permission card is still shown until I leave/reload the screen.

This code triggers on the scan QR Code fragment, but the user returns to the `CheckInsFragment`, we need to re-check on resume there.

Reply:
I don't think the issue is here nor with using `shouldShowRequestPermissionRationale ` in `CheckInsFragment `
Please check 4d1581e9da62dfba5354f49f81e009e4c13ffd18